### ISSUE TITLE
Fixed dependency causes installation issues

### ISSUE THREAD

**Issue by pneff** [dajiaji/pyseto#333]
**Describe the bug**

pyseto currently has fixed version requirements for cryptography and pycryptodomex. The cryptography dependency at least is preventing us from installing pyseto into an environment where we also have certain other packages with different requirements. In our case the conflict is with `pyopenssl`.

Would you consider relaxing the requirement to use `>=` instead of specific version numbers? That should make it easier to use pyseto as a library on larger environments.

If you agree with this approach I am happy to help with providing a pull request.

**To Reproduce**

1. Create a new virtualenv: `python -m venv /tmp/testenv`
2. Activate the virtualenv: `source /tmp/testenv/bin/activate`
3. Ensure I'm using the latest pip: `pip install --upgrade pip`
4. Install the latest versions of `pyopenssl` and `pyseto`: `pip install 'pyopenssl>=23.3.0' 'pyseto>=1.7.5'`

Output:

```
Collecting pyopenssl>=23.3.0
  Downloading pyOpenSSL-23.3.0-py3-none-any.whl.metadata (12 kB)
Collecting pyseto>=1.7.5
  Downloading pyseto-1.7.5-py3-none-any.whl.metadata (17 kB)
Collecting cryptography<42,>=41.0.5 (from pyopenssl>=23.3.0)
  Downloading cryptography-41.0.5-cp37-abi3-manylinux_2_28_x86_64.whl.metadata (5.2 kB)
Collecting argon2-cffi<24.0.0,>=23.1.0 (from pyseto>=1.7.5)
  Downloading argon2_cffi-23.1.0-py3-none-any.whl.metadata (5.2 kB)
INFO: pip is looking at multiple versions of pyseto to determine which version is compatible with other requirements. This could take a while.
ERROR: Cannot install pyopenssl==23.3.0 and pyseto==1.7.5 because these package versions have conflicting dependencies.

The conflict is caused by:
    pyopenssl 23.3.0 depends on cryptography<42 and >=41.0.5
    pyseto 1.7.5 depends on cryptography==41.0.4

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts
```

Python version is at 3.11.5 and PIP at 23.3.1.

**Comment by dajiaji** [comment]
thanks for the report. I will check and fit it.

**Comment by dajiaji** [comment]
@pneff I've released v1.7.6 to fix it. Can you please confirm that the issue has been resolved?

**Comment by pneff** [comment]
Hi @dajiaji. I appreciate your super-quick response on this ticket. The issue does indeed look resolved for me!
